components-api/src/main/scala/pl/touk/nussknacker/engine/api/generics/MethodTypeInfo.scala

```diff
@@ -1,10 +1,16 @@
 package pl.touk.nussknacker.engine.api.generics
 
-import pl.touk.nussknacker.engine.api.typed.typing.{Typed, TypedClass, TypingResult}
+import pl.touk.nussknacker.engine.api.typed.typing.{Typed, TypedClass, TypingResult, Unknown}
 
 object MethodTypeInfo {
   private val arrayClass = classOf[Array[Object]]
 
+  val noArgTypeInfo = MethodTypeInfo(
```

Comment:
To be precise it is `noArgReturningUnknownTypeInfo`. I would add a return type as a parameter and reuse it also in boolean conversion.

Reply:
Done